### ISSUE TITLE
Emulator does not render valid adaptive card in an activity

### ISSUE THREAD

**Issue by chrimc62** [microsoft/BotFramework-Emulator#2154]
Version of emulator: 4.9.1

The below activity has a valid adaptive card that renders properly in adaptivecards.io.  In emulator it looks like the below.  It seems related to having a curly bracest in the string value.  If I change it to obj= and remove the trailing curly brace it works fine.

In emulator (wrong):
![image](https://user-images.githubusercontent.com/17637681/83312085-14a25180-a1c6-11ea-9923-1589d8c91dbe.png)

In adaptive cards designer (right):
![image](https://user-images.githubusercontent.com/17637681/83312252-7ebaf680-a1c6-11ea-8b89-8b711c985691.png)

Activity:
```
{
    "type": "message",
    "attachments": [
        {
            "contentType": "application/vnd.microsoft.card.adaptive",
            "content": {
                "type": "AdaptiveCard",
                "version": "1.0",
                "body": [
                    {
                        "type": "TextBlock",
                        "text": "simple",
                        "size": "large"
                    },
                    {
                        "type": "FactSet",
                        "facts": [
                            {
                                "title": "due",
                                "value": "{timex: [2020-06-01], type: date}"
                            }
                        ]
                    }
                ],
                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
            }
        }
    ]
}
```

**Comment by tonyanziano** [comment]
We should test this in Web Chat.

**Comment by stevkan** [comment]
Possibly related to Web Chat bug [#3165](https://github.com/microsoft/BotFramework-WebChat/issues/3165)

**Comment by chrimc62** [comment]
This is still an issue.  This: 
![image](https://user-images.githubusercontent.com/17637681/92188263-f8d7c880-ee10-11ea-8413-ebbca8cf3f0f.png)
Renders as this:
![image](https://user-images.githubusercontent.com/17637681/92188277-02613080-ee11-11ea-9bf1-3bdabe5ef081.png)

**Comment by tonyanziano** [comment]
I just tested this with the latest Web Chat and the bug is still present in 4.10.0.

The issue (https://github.com/microsoft/BotFramework-WebChat/issues/3165) has been labeled for R11 and can be tracked there.

**Comment by chrimc62** [comment]
This was not fixed in R11 or R12.  Is it ever going to be fixed?

**Comment by tonyanziano** [comment]
@chrimc62 this is a Web Chat bug so it will be present in Emulator until it is fixed in Web Chat. If you look at https://github.com/microsoft/BotFramework-WebChat/issues/3165 you can see that it was moved into the R13 milestone last week.

**Comment by corinagum** [comment]
In progress now - sorry @chrimc62 for the wait. Low on manpower haha.

**Comment by amazingYolk** [comment]
Still not fixed in version  4.14.1 of emulator, how's it going?

**Comment by amazingYolk** [comment]
> Still not fixed in version 4.14.1 of emulator, how's it going?

![image](https://user-images.githubusercontent.com/42231796/173322884-41d5a89b-dc78-456e-baf3-9cd84210d117.png)
